src/main/java/org/skriptlang/skript/bukkit/recipes/elements/ExprLastRecipe.java

```diff
@@ -0,0 +1,56 @@
+package org.skriptlang.skript.bukkit.recipes.elements;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.Recipe;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Last Registered Recipe")
+@Description("Gets the recipe that was most recently registered through the Register Recipe Section.")
+@Examples({
+	"register a new shapeless recipe with the key \"my_recipe\":",
+		"\tset ingredients to diamond, emerald and iron ingot",
+		"\tset recipe result to diamond sword",
+	"set {_last} to last registered recipe"
+})
+@Since("INSERT VERSION")
+public class ExprLastRecipe extends SimpleExpression<Recipe> {
```

Comment:
I don't think this is necessary. Recipes have keys that you can easily use to get the recipe you want. I think we should encourage that instead of time-based selectors.